torch/_functorch/aot_autograd.py

```diff
@@ -771,11 +771,16 @@ def _dup_fake_script_obj(fake_flat_args):
                 != 0
             ):
                 raise RuntimeError(
-                    f"""\
+                    """\
 Found a graph input that requires gradients, and received a mutation.
-This is currently banned in the aot_export workflow. If you need this functionality, please file a github issue.
+This is currently banned in the aot_export workflow. If you need this functionality,
+please file a github issue and submit the trace log.
 
-fw_metadata={str(fw_metadata)}"""
+Get trace log by running with `TORCH_TRACE`:
+
+    TORCH_TRACE="/tmp/tracedir" python foo.py
+
+or following the insturction https://pytorch.org/docs/stable/torch.compiler_troubleshooting.html#tlparse-torch-trace"""
```

Comment:
are you sure that running with TORCH_TRACE will actually log the `fw_metadata` under this code path? I think we should:

(1) try running a repro with `TORCH_TRACE=...` turned on and confirm whether fw_metadata actually shows up in the tlparse output

(2) if not, you can call `trace_structured right before this error to manually log it